frontend/src/views/CensusDirectory/components/Project/connect.ts

```diff
@@ -1,26 +1,54 @@
-import { getProjectTier } from "../../utils";
+/* eslint-disable sonarjs/cognitive-complexity */
 import { ProjectProps } from "./types";
 import notebookLinks from "census-notebook-links.json";
 
-export const useConnect = ({ project, id }: ProjectProps) => {
-  const date = new Date(
-    project.last_updated || project.submission_date || ""
-  ).toLocaleDateString("en-US", {
+export const useConnect = ({ clobberedProjects }: ProjectProps) => {
+  const sharedProject = clobberedProjects[0];
+
+  let date = new Date(
+    sharedProject.last_updated || sharedProject.submission_date || 0
+  );
+
+  if (date === new Date("1970-01-01")) {
```

Comment:
should we do:

```ts
const DEFAULT_EPOCH_TIME = 0;
date.getTime() === DEFAULT_EPOCH_TIME
```

It seems more readable and less risky than dealing with timezone stuff, since: `new Date("1970-01-01")` returns
`Wed Dec 31 1969 16:00:00 GMT-0800 (Pacific Standard Time)`